### ISSUE TITLE
Content Element Relations broken after Upgrade to 12

### ISSUE THREAD

**Issue by SicorDev** [Gernott/mask#583]
Hello there,

So I upgraded a page from TYPO3 10 to 11 to 12. Latest core, mask and PHP 8.2.

On checking the masks I noticed that the ones using content elements where all broken. In the old database only the parent Id was stored, which was split into three fields now: 

tx_mask_content_parent_uid, tx_mask_content_role, tx_mask_content_tablenames

I figured out to repair those relations manually, both in tt_content and in the loop tables, but I wonder if that couldn't better be handled by an upgrade wizard?

Greetings,
Manuel

**Comment by nhovratov** [comment]
Hi Manuel, there is an Upgrade Wizard for that: https://docs.typo3.org/p/mask/mask/main/en-us/ChangeLog/8.1/Index.html#upgrade-wizard-necessary

**Comment by SicorDev** [comment]
I swear I didn't see that one during the update process on 11 or 12 in the upgrade module. Or is it commandline only?

**Comment by nhovratov** [comment]
> Or is it commandline only?
No, this is a real Upgrade Wizard. If it finds at least one field of type "Content", the Upgrade Wizard will appear:
https://github.com/Gernott/mask/blob/main/Classes/Updates/MigrateContentFields.php
Maybe the Mask configuration was missing during the Upgrade Wizards steps?

**Comment by SicorDev** [comment]
Hm... that's a good question. It was missing a while before I moved json + templates from mask_project to the site package and I'm not sure if I checked for wizards again after that. Also I did that on 12 only, When coming from 10 I rush quite fast through 11, not much checking or repairing stuff there... :-)

I'll do another upgrade with referenced Content Elements, where i can observe it more closely. Thanks for the info so far :)
